docs/source/starter/lightning_lite.rst

```diff
@@ -70,6 +70,8 @@ The ``run`` function contains custom training loop used to train ``MyModel`` on
     run(args)
 
 
+Checkout MNIST training example with :class:`~pytorch_lightning.lite.LightningLite`  `here <https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/mnist_examples/image_classifier_2_lite.py>`_
+
```

Comment:
This is out of place here. I don't think we should point to a full Lite example if we are about to show how to convert in the next section. Can this be moved down to the end of the "Converting to LightningLite" section?

```suggestion
Check out this MNIST training example with :class:`~pytorch_lightning.lite.LightningLite`  `here <https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/mnist_examples/image_classifier_2_lite.py>`_

```

Reply:
thanks for the suggestion @awaelchli! I have Updated the PR.